styles/fonts.scss

```
@@ -60,9 +60,9 @@ $font-base-style: normal;
 $font-base-variant: normal;
 
 @font-face {
-  font-family: 'Visby';
-  src: url('~/public/assets/fonts/VisbyCF-Regular.otf');
-  font-weight: 400;
+  font-family: 'Visby' serif;
+  src: url('~/public/assets/fonts/VisbyRegular.otf');
```

```suggestion
  src: url('/assets/fonts/VisbyRegular.otf');
```
Change font import from `assets` route

---

my fault. I will solve it. @luleheco22 don't change this file